start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+DIR="data"
+
+if ! [ -d "$DIR" ]; then
+    mkdir $DIR
+fi
+
+docker run --rm --user="$(id -u):$(id -g)" --name=scraper \
+           -v $PWD/data:/app/scraper/data \
```

Comment:
Put `$PWD` (or even better, the whole argument) in quotes `"` to avoid problems if the path contains a space